core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/dot-uve-toolbar.component.ts

```diff
@@ -1,16 +1,29 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
 
 import { ButtonModule } from 'primeng/button';
 import { ToolbarModule } from 'primeng/toolbar';
 
-import { DotMessagePipe } from '@dotcms/ui';
+import { TOOLBAR_VARIANTS } from '../../../shared/enums';
+import { UVEStore } from '../../../store/dot-uve.store';
+import { DotEmaBookmarksComponent } from '../dot-ema-bookmarks/dot-ema-bookmarks.component';
+import { DotEmaInfoDisplayComponent } from '../dot-ema-info-display/dot-ema-info-display.component';
 
 @Component({
     selector: 'dot-uve-toolbar',
     standalone: true,
-    imports: [ButtonModule, DotMessagePipe, ToolbarModule],
+    imports: [ButtonModule, ToolbarModule, DotEmaBookmarksComponent, DotEmaInfoDisplayComponent],
     templateUrl: './dot-uve-toolbar.component.html',
     styleUrl: './dot-uve-toolbar.component.scss',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class DotUveToolbarComponent {}
+export class DotUveToolbarComponent {
+    #store = inject(UVEStore);
+
+    readonly $toolbar = this.#store.$uveToolbar;
+
+    TOOLBAR_VARIANTS = TOOLBAR_VARIANTS;
```

Comment:
Mark this with a comment so we remember to delete this when we migrate to the new toolbar or you can also use the FF, we have it in the store as `$previewMode` maybe we can change that name to something more meaningful

Reply:
true, i remove the enum and only use the previewMode from the store now